docs/BreakingChanges/168 DataContractJsonSerializer incompatible with ES6 (and V8).md

```diff
@@ -0,0 +1,35 @@
+## 168: DataContractJsonSerializer incompatible with ES6 (and V8)
+
+### Scope
+Edge
+
+### Version Introduced
+4.7
+
+### Source Analyzer Status
+Planned
+
+### Change Description
+Starting in .NET Framework .NET 4.7, the compatibility issue between the escaped strings for some special control characters generated by DataContractJsonSerializer and ECMAScript V6 has been fixed. 
+
+- [x] Quirked
+- [ ] Build-time break
+
+### Recommended Action
+Prior to .NET 4.7, this change is disabled by default. You can enable this fix by adding the following line to the `<runtime>` section of the app.config or web.config file:
```

Comment:
According to DevDiv #378461, this is a retargeting feature. So this should read:
For apps that target the .NET Framework 4.7, this feature is enabled by default. If this behavior is not desirable, you can opt out of this feature by adding the following line to the `<runtime section of the `app.config` or `web.config` file:
```xml
<runtime> 
    <AppContextSwitchOverrides value="Switch.System.Runtime.Serialization.DoNotUseECMAScriptV6EscapeControlCharacter=true" /> 
</runtime>
```
Apps that target the .NET Framework 4.6.2 and earlier versions but are running under the .NET Framework 4.7 or later can opt into this behavior y adding the following line to the `<runtime>` section of the `app.config` or `web.config` file: